packages/manager/apps/public-cloud/src/assets/theme/_sidebar.less

```diff
@@ -47,6 +47,7 @@ cloud-sidebar { /* stylelint-disable-line */
     background-color: @primary-color;
     color: @light-color;
     top: 0;
+    left: 0;
```

Comment:
Is the `left` property required?

It was previously updated with a `transform: translateX(0);` and also transitions are made with `.ng-enter|ng-enter-active|ng-leave|ng-leave-active`.

Reply:
It's needed for mobile display, I can add a media query if necessary